R/get_kr_comp.R

```diff
@@ -0,0 +1,115 @@
+#' obtain kr component
```

Comment:
Please use conventional formatting for documentation, in particular title style for title, see other roxygen chunks for reference (also blank lines to make it easier to read)